corehq/apps/reports/views.py

```diff
@@ -957,6 +957,19 @@ def rebuild_case_view(request, domain, case_id):
 @require_case_view_permission
 @require_permission(Permissions.edit_data)
 @require_POST
+def resave_case(request, domain, case_id):
+    case = get_document_or_404(CommCareCase, domain, case_id)
+    CommCareCase.get_db().save_doc(case._doc)  # don't just call save to avoid signals
```

Comment:
@gcapalbo sending to elastic is the primary way i expect to use this, but if you think there's a chance it will ever be useful for reminders then seems like we could easily change it. i'm not too worried about the overhead since it's not something that will happen in bulk